plugins/node-team/scripts/lib/api/issue.sh

```
@@ -0,0 +1,147 @@
+        with open(rf) as f:
+            data = json.load(f)
+        for name, github in data.get('members', {}).items():
+            if query in name.lower():
+                print(json.dumps({'displayName': name, 'github': github, 'source': os.path.basename(rf)}))
+    except Exception:
+        continue
+" "$query" "$root_dir"
+
+  # Also search via Jira API
+  local encoded
+  encoded=$(_jql_encode "$query")
```

Shellcheck (0.11.0)

[warning] encoded appears unused. Verify use (or export if used externally).

(SC2034)